esmvaltool/utils/recipe_test_workflow/app/install_env_file/rose-app.conf

```
@@ -1,11 +1,14 @@
-# Copy the site specific environment file to the 'share/bin' directory in the
-# installed Cylc workflow (this directory is automatically added to the
-# ${PATH} by Cylc).
+# Copy the site specific environment file
+# to the 'share/bin' directory in the installed Cylc workflow
+# (this directory is automatically added to the ${PATH} by Cylc).
+#
+# BASH_XTRACEFD=1 sends the output from 'set -x'
+# to 'stdout' rather than 'stderr'.
 
 [command]
-default=BASH_XTRACEFD=1  # Send the output from 'set -x' to 'stdout' rather than 'stderr'.
+default=BASH_XTRACEFD=1
        =set -euxo pipefail
-       =cp ${ENV_FILE_SITE_PATH} ${SHARE_BIN_DIR}/${ENV_FILE}
+       =cp ${ENV_FILE_SITE_PATH} ${SHARE_BIN_DIR}/.
```

```suggestion
       =cp ${ENV_FILE_SITE_PATH} ${SHARE_BIN_DIR}/
```
Similar suggestion to the one in `opt/rose-app-dkrz.conf`.